lua/nordic/utils.lua

```diff
@@ -11,28 +11,41 @@ function M.highlight(table)
     end
 end
 
+function M.none()
+    return 'NONE'
+end
+
 function M.is_none(string)
     return string == 'NONE' or string == 'none'
 end
 
-function M.none()
-    return 'NONE'
+function M.is_table(value)
+    return type(value) == 'table'
 end
 
 function M.merge(table1, table2)
-    if table1 == table2 == nil then return {} end
-    if table1 == nil then
-        return table2
-    elseif table2 == nil then
+    -- TODO what about replacing this with merge_inplace
+    if not table1 then
+        return table2 or {}
+    elseif not table2 then
         return table1
     end
     return vim.tbl_deep_extend('force', table1, table2)
 end
```

Comment:
What do you think about removing this function altogether? I need `merge_inplace` anyway, so I think it would make sense to just use that everywhere. @AlexvZyl

Reply:
Meh, I'm just going to do it... ¯\\\_(ツ)_/¯ 

Reply:
See my review.